src/openrct2/localisation/StringIds.h

```diff
@@ -3855,6 +3855,9 @@ enum : uint16_t
     STR_OPTIONS_TOOLBAR_BUTTONS_CENTRED = 6629,
     STR_OPTIONS_TOOLBAR_BUTTONS_CENTRED_TIP = 6630,
 
+    STR_CHEAT_IGNORE_PRICE = 6631,
+    STR_CHEAT_IGNORE_PRICE_TIP = 6632,
+
```

Comment:
As these are UI exclusive strings, these should now go into the new UiStringIds.h file in the openrct2-ui subproject.

Reply:
I have moved the strings over.